build_tools/benchmarks/README.md

```diff
@@ -0,0 +1,49 @@
+# IREE Benchmark Suites Tool
+
+This directory contains the tools to run IREE benchmark suites and generate
+reports. More information about benchmark suites can be found [here](benchmarks/README.md).
+
+## Benchmark Tools
+Currently we have `run_benchmarks_on_android.py` and `run_benchmarks_on_linux.py` to run
+benchmark suites on Android devices (with `adb`) and Linux local machines.
+
+The available arguments can be shown with `--help`. Here we list some common
+usages (assume the target `iree-benchmark-suites` is built under `$IREE_BUILD`):
+
+**Run all benchmarks**
+```sh
+./run_benchmarks_on_linux.py \
+  --normal_benchmark_tool_dir=$IREE_BUILD/iree/tools \
+  --output=results.json $IREE_BUILD
+```
+
+**Run all benchmarks and capture the tracy files**
+```sh
+./run_benchmarks_on_linux.py \
+  --normal_benchmark_tool_dir=$IREE_BUILD/iree/tools \
+  --traced_benchmark_tool_dir=$IREE_TRACED_BUILD/iree/tools \
+  --trace_capture_tool=${path of iree-tracy-capture} \
+  --capture_tarball=captured_tracy_files.tar
+  --output=results.json $IREE_BUILD
+```
+
+**Run selected benchmarks with the filters**
+```sh
+./run_benchmarks_on_linux.py \
+  --normal_benchmark_tool_dir=$IREE_BUILD/iree/tools \
+  --model_name_regex="MobileBertSquad" \
+  --driver_filter_regex="dylib" \
+  --mode_regex="4-threads" \
+  --output=results.json $IREE_BUILD
+```
+
+## Generate Benchmark Report
+The tools here are mainly designed for benchmark automation pipeline.
+The `post_benchmarks_as_pr_comment.py` and `upload_benchmarks_to_dashboard.py`
+are used to upload and post reports to the dashboard.
```

Comment:
grammar/style nits

* newline between header and content
* plural agreement for "pipelines"
* add "scripts" ("files" would also work)

```suggestion
## Generate Benchmark Report

The tools here are mainly designed for benchmark automation pipelines.
The `post_benchmarks_as_pr_comment.py` and `upload_benchmarks_to_dashboard.py`
scripts are used to upload and post reports to the dashboard.
```